entrypoint

```diff
@@ -4,42 +4,66 @@ if [ -n "$DEBUG" ]; then
     set -x
 fi
 
-if ! find . -mindepth 1 | read -r; then
-    >&2 echo "Creating default configs..."
-    cp -r /opt/cfx-server-data/* /config
-    RCON_PASS="${RCON_PASSWORD-$(tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 16)}"
-    sed -i "s/{RCON_PASS}/${RCON_PASS}/g" /config/server.cfg;
-    >&2 echo "----------------------------------------------"
-    >&2 echo "RCON password is set to: ${RCON_PASS}"
-    >&2 echo "----------------------------------------------"
-fi
+# Empty config args defaults to loading txAdmin
+CONFIG_ARGS=
 
-if [ -z "$NO_ONESYNC" ]; then
-    ONESYNC_ARGS="+set onesync on +set onesync_population true"
-fi
+# Temporary during the transition - this way old configs still trigger the old (no txadmin by default) behaviour.
+NO_TXADMIN = "DONTANSWERTHEPHONETOLESTERCREST"
```

Comment:
This needs to go in the Dockerfile so it doesn't override user overrides

Reply:
Done!